src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Builder/IScaffoldBuilder.cs

```diff
@@ -29,6 +29,14 @@ public interface IScaffoldBuilder
     /// </summary>
     IScaffoldBuilder WithOption(ScaffolderOption option);
 
+    /// <summary>
+    /// Adds mulitple options to be used with this scaffolder. This adds the configured options to both the command line and
+    /// to the dotnet-scaffold interactive UI.
+    /// </summary>
+    /// <param name="options"></param>
+    /// <returns></returns>
+    IScaffoldBuilder WithOptions(List<ScaffolderOption> options); 
```

Comment:
I think I agree that is a better way to expose this so i'll change it to `params IEnumerable<ScaffolderOption>` for now and then after API review, we can edit it then

Reply:
Ah, VS didn't complain when I tried it, but you're right when I build it it tells me it needs preview language. `params string[]` would be the current version alternative then. Hrm. `IEnumerable<string>` is fine for now since we have collection initializers. Before those I would have said to go with param string[] because the syntax we'd be forcing the user to use to create a new collection each time was too verbose. Need to remember to revisit this though